### ISSUE TITLE
Create services diagram

### ISSUE THREAD

**Issue by berkaycubuk** [ProjectSOTAS/documents#36]
Create services diagram and add it to documentation.

Services diagram will explain what kind of services we'll use and how they'll interact with each other.

**Comment by berkaycubuk** [comment]
![sotas-project-architecture](https://user-images.githubusercontent.com/29932133/140606609-91c450b2-5ecc-4667-987c-2e0d469da6c6.png)

**Comment by First008** [comment]
So there is no API Gateway right? Is that really the right decision? I dont know. Other than that it looks ok to me.

**Comment by berkaycubuk** [comment]
Yes we not gonna use gateway if needed in the future we can add. 

Today I did a deep research about Cassandra and it's not a great idea to create database for each service. Cassandra is powerful enough to store and manage all of our data inside one db.

Also we'll need an event stream to prevent system crash due to overload. That's also a reason to not separate Cassandra.

**Comment by First008** [comment]
Ok then!
